web_src/less/_repository.less

```diff
@@ -1614,6 +1614,18 @@
       margin-right: .25rem;
     }
 
+    .diff-detail-stats strong:nth-of-type(1) {
+      color: var(--color-yellow);
+    }
+
+    .diff-detail-stats strong:nth-of-type(2) {
+      color: var(--color-green);
+    }
+
+    .diff-detail-stats strong:nth-of-type(3) {
+      color: var(--color-red);
+    }
```

Comment:
Hmm… I don't think it's a good idea to style the `n-th strong` specially.
What about adding special `diff-stats-changed-files`, `diff-stats-additions`, and `diff-stats-deletions` classes (or similar) instead?

Reply:
I would need to edit every single translation, because the translations contain the html <strong> part.

Reply:
Okay, then please just add a comment above this CSS, that's also fine in this case.